packages/cli/src/utils/platform.ts

```diff
@@ -71,7 +71,7 @@ export const platformRequestOrThrow = async (params: {
   token: string
   path: string
   payload?: object
-}) => {
+}): Promise<object> => {
```

Comment:
@jasonkuhrt lets re-iterate on this one. I think we could just remove explicit return type and avoid casting `as Promise<any>`

I will approve and merge this PR as it is.